CHANGELOG.md

```diff
@@ -34,15 +34,16 @@
 
 ### Improvements
 
-- Improve e2e tests reliability ([#2580](https://github.com/kedacore/keda/issues/2580))
+- **General:** Fix generation of metric names if any of ScaledObject's triggers is unavailable ([#2592](https://github.com/kedacore/keda/issues/2592))
+- **Azure Queue:** Don't call Azure queue GetProperties API unnecessarily ([#2613](https://github.com/kedacore/keda/pull/2613))
 
 ### Breaking Changes
 
 - TODO ([#XXX](https://github.com/kedacore/keda/issue/XXX))
 
 ### Other
 
-- TODO ([#XXX](https://github.com/kedacore/keda/issue/XXX))
+- **General:** Improve e2e tests reliability ([#2580](https://github.com/kedacore/keda/issues/2580))
```

Comment:
Have you changed this from `### Improvements` to `### Other` because of any reason?
It's not a blocker, just to know it

Reply:
Yes. I saw that for 2.6.1 such an item was under 'Other' (Fix failng tests...) so wanted it to be consistent